src/conditionalAndLoops.js

```diff
@@ -69,7 +88,12 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a, b, c) {
-	throw new Error("Not implemented");
+	if ((a + b > c && b + c > a && a + c > b)) {
+		return true;
+	}
+	else {
+		return false;
+	}
```

Comment:
Try returning the value of the conditonal directly. You don't need to compare boolean values before returning them.

Reply:
Oh ha bhaiya. Aisa:?
return (a + b > c && b + c > a && a + c > b);
Thanks bhaiya

Reply:
Yes